units/host_monitoring_idle_termination_test.go

```diff
@@ -68,6 +69,7 @@ func TestFlaggingIdleHosts(t *testing.T) {
 			// creation time would otherwise indicate it has been idle a while
```

Comment:
Is there a test you can add for the new behavior somewhere in this file?

Reply:
Yes - I've add some new tests